src/core/router/net_db/impl.h

```diff
@@ -197,16 +197,16 @@ class NetDb : public NetDbTraits {
 
   std::shared_ptr<const RouterInfo> GetClosestFloodfill(
       const IdentHash& destination,
-      const std::set<IdentHash>& excluded) const;
+      const std::set<IdentHash>& excluded);
 
   std::vector<IdentHash> GetClosestFloodfills(
       const IdentHash& destination,
       std::uint8_t num,
-      std::set<IdentHash>& excluded) const;
+      std::set<IdentHash>& excluded);
 
   std::shared_ptr<const RouterInfo> GetClosestNonFloodfill(
       const IdentHash& destination,
-      const std::set<IdentHash>& excluded) const;
+      const std::set<IdentHash>& excluded);
```

Comment:
These getters should remain const. They should only be accessing members, not mutating.

Reply:
> These getters should remain const

I thought so as well, but didn't know how much work would be needed on the dispatcher to make it const as well. Will work on this.